service-base/src/main/java/fi/nls/oskari/domain/map/UserDataLayer.java

```diff
@@ -76,4 +77,10 @@ public WFSLayerOptions getWFSLayerOptions() {
         }
         return options;
     }
+    // FIXME: used to populate localized name for new layers, remove when frontend sends locale object.
+    public void setLocalizedNames (String name) {
+        for (String lang : PropertyUtil.getSupportedLanguages()){
+            setName(lang, name);
+        }
+    }
```

Comment:
Call https://github.com/oskariorg/oskari-server/blob/master/service-base/src/main/java/fi/nls/oskari/domain/map/JSONLocalizedName.java#L10 instead with instance default language so we don't need to add temporary methods.